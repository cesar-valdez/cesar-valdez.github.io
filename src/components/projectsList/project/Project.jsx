import React from 'react';

import './project.scss';

const Project = (props) => {
    return (
      <li className="pi_Card-projectsList-li">
				<a href={props.link} target="_blank">
					<span className="pi_Card-projectsList-li-project">{props.name} </span>
					<span> - </span>
					<span className="pi_Card-projectsList-li-description">{props.description}</span>
				</a>
			</li>
    )
}

export default Project;