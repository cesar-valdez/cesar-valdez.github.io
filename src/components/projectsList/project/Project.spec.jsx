import React from 'react';
import { shallow } from 'enzyme';

import Project from './Project';

describe('Project', () => {
    const project = shallow(<Project />);

    it('should render Project component', () => {
        expect(project).toMatchSnapshot();
    });
});