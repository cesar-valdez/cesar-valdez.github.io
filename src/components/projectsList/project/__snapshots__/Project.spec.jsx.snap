// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project should render Project component 1`] = `
<li
  className="pi_Card-projectsList-li"
>
  <a
    target="_blank"
  >
    <span
      className="pi_Card-projectsList-li-project"
    >
       
    </span>
    <span>
       - 
    </span>
    <span
      className="pi_Card-projectsList-li-description"
    />
  </a>
</li>
`;
